front/components/app/MainTab.jsx

```diff
@@ -32,8 +33,18 @@ export default function MainTab({ app, currentTab, user, readOnly }) {
       ),
     },
     {
-      name: "Runs",
+      name: "Logs",
       href: `/${user}/a/${app.sId}/runs`,
+      icon: (
+        <ArchiveBoxArrowDownIcon
+          className="h-4 w-4 flex-shrink-0 mr-2 mt-0.5"
+          aria-hidden="true"
+        />
+      ),
+    },
+    {
+      name: "Execute",
```

Comment:
Should we say “run” maybe now that runs is named logs ?

Reply:
Sounds good to me

Reply:
Changed the display aspect only -- endpoints are clashing with each-other if I change it to `run`. Would require changing the existing `runs` path as well.
Happy to do it if that's safe